pubsub/kafka/kafka.go

```diff
@@ -62,6 +65,46 @@ func (p *PubSub) Subscribe(ctx context.Context, req pubsub.SubscribeRequest, han
 	return p.kafka.Subscribe(p.subscribeCtx)
 }
 
+func (p *PubSub) BulkSubscribe(ctx context.Context, req pubsub.SubscribeRequest, handler pubsub.BulkHandler) error {
+	p.kafka.AddTopicBulkHandler(req.Topic, adaptBulkHandler(handler))
+	maxBulkCount, err := strconv.Atoi(req.Metadata["maxBulkCount"])
+	if err != nil {
+		maxBulkCount = 20
+	}
+	maxBulkLatencyMilliSeconds, err := strconv.Atoi(req.Metadata["maxBulkLatencyMilliSeconds"])
+	if err != nil {
+		maxBulkLatencyMilliSeconds = 20
```

Comment:
Defaults should be updated, maybe even have a common default for all components.

Reply:
Done